pages/docs/reference/whatsnew14.md

```diff
@@ -620,6 +620,41 @@ we also added `all-compatibility` mode.
 For more information about default methods in the Java interop, see the [documentation](java-to-kotlin-interop.html#default-methods-in-interfaces) and 
 [this blog post](https://blog.jetbrains.com/kotlin/2020/07/kotlin-1-4-m3-generating-default-methods-in-interfaces/). 
 
+### Type annotations in the JVM bytecode
+
+Kotlin now can generate type annotations in the JVM bytecode (target version 1.8+), so that they become available at runtime.
```

Comment:
```suggestion
Kotlin can now generate type annotations in the JVM bytecode (target version 1.8+), so that they become available in Java reflection at runtime.
```